### PR TITLE
fix #177: [CA] Fix handling of the 'terme' template

### LINKED ISSUE #177: [CA] Fix handling of the 'terme' template
- Wiktionary page: https://ca.wiktionary.org/wiki/disset

Wikicode:
```
{{terme|it|come}}
```

Output:
```
come
```

Expected:
```
<i>come</i>
```

---

Model link, if any: https://ca.wiktionary.org/wiki/Plantilla:terme

### PR DESCRIPTION
**Pull request by lasconic**
- [x] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

File: tests/test_ca.py [hunk 1]
```diff
@@ -192,8 +192,22 @@ def test_parse_word(word, pronunciation, genre, definitions, page):
         ("{{marca-nocat|ca|occidental|balear}}", "<i>(occidental, balear)</i>"),
         ("{{q|tenir bona planta}}", "<i>(tenir bona planta)</i>"),
         ("{{q|{{m}}}}", "<i>(m.)</i>"),
-        ("{{terme|it|come}}", "come"),
-        ("{{terme|ca|seu|el seu}}", "el seu"),
+        ("{{terme|it|come}}", "<i>come</i>"),
```
Comment (BoboTiG) [inline tests/test_ca.py:195]: You should move all those tests into doctests of `last_template_handler()`.

File: scripts/lang/ca.py [hunk 2]
```diff
@@ -85,15 +87,53 @@
...
 # Templates that will be completed/replaced using custom style.
 templates_other = {
     "m": "m.",
 }
 
+
+def last_template_handler(template: Tuple[str, ...], locale: str) -> str:
+    """
+    Will be called in utils.py::transform() when all template handlers were not used.
+        >>> last_template_handler(["terme", "it", "come"], "ca")
```
Comment (BoboTiG) [inline scripts/lang/ca.py:101]: Add a blank line to separate the description from doctests.

**Review by BoboTiG — CHANGES_REQUESTED** [review]
Almost perfect 💪

File: tests/test_ca.py (hunk 1 above)
Comment (lasconic) [reply tests/test_ca.py]: OK. Done.

File: scripts/lang/ca.py (hunk 2 above)
Comment (lasconic) [reply scripts/lang/ca.py:102]: OK. Done.

**Review by BoboTiG — APPROVED** [review]

**Comment by BoboTiG** [comment]
Thanks a lot @lasconic  :)

**Comment by BoboTiG** [comment]
@all-contributors please add @lasconic

**Comment by BoboTiG** [comment]
@all-contributors please add @lasconic for code

**Pull request closed without merging**